.travis.yml

```diff
@@ -91,11 +91,20 @@ before_install:
     - pwd
     - mkdir /tmp/cozy-desktop
     - ln -s /tmp/cozy-desktop tmp
+    - npm install -g yarn
```

Comment:
Travis will install yarn but you still have to install the modules in gui manually